jobs/release.Jenkinsfile

```diff
@@ -96,11 +96,11 @@ podTemplate(cloud: 'openshift', label: pod_label, yaml: pod) {
             """)
         }
 
-        def basearches = ""
-        if (params.ARCHES == '') {
-            basearches = shwrapCapture("jq -r '.builds | map(select(.id == \"${params.VERSION}\"))[].arches[]' builds/builds.json").split()
-        } else {
-            basearches = params.ARCHES.split()
+        def basearches = params.ARCHES.split()
+        def builtarches = shwrapCapture("jq -r '.builds | map(select(.id == \"${params.VERSION}\"))[].arches[]' builds/builds.json").split()
+        if builtarches != basearches {
```

Comment:
This comparison might need to be improved to make it not depend on ordering.

Reply:
I think if we convert it to a set instead it should work.

Reply:
I converted the logic to use sets using the `.toSet()` function which I think should work???